packages/common/src/types/gethGenesis.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Interface for Geth Genesis Config
+ */
+export interface GethGenesisConfig {
+  chainId: number
+  homesteadBlock?: number
+  daoForkBlock?: number
+  daoForkSupport?: boolean
+  eip150Block?: number
+  eip150Hash?: string
+  eip155Block?: number
+  eip158Block?: number
+  byzantiumBlock?: number
+  constantinopleBlock?: number
+  petersburgBlock?: number
+  istanbulBlock?: number
+  muirGlacierBlock?: number
+  berlinBlock?: number
+  londonBlock?: number
+  mergeForkBlock?: number
+  cancunBlock?: number
+  shanghaiTime?: number
+  cancunTime?: number
+  terminalTotalDifficulty?: number
+  terminalTotalDifficultyPassed?: boolean
+  ethash?: {}
+  clique?: {
+    period?: number
+    epoch?: number
+    blockperiodseconds?: number
+    epochlength?: number
+  }
+  trustedCheckpoint?: {
+    sectionIndex: number
+    sectionHead: string
+    chtRoot: string
+    bloomRoot: string
+  }
+  trustedCheckpointOracle?: {
+    address: string
+    signers: string[]
+    threshold: number
+  }
```

Comment:
In Hoodi there's also a `depositContractAddress` param https://github.com/eth-clients/hoodi/blob/7f71638afc431675c19b9e306998b9d4925b7cec/metadata/genesis.json#L31

Reply:
Indeed, added that also